atomic-reactor.spec

```diff
@@ -38,7 +38,8 @@ Summary:        Improved builder for Docker images
 Group:          Development/Tools
 License:        BSD
 URL:            https://github.com/%{owner}/%{project}
-Source0:        https://github.com/%{owner}/%{project}/archive/%{commit}/%{project}-%{commit}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```

Comment:
koji-containerbuild uses this:

```
%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
```

The `-XXXXXX` is odd, but honestly I have no idea which one is better.